format/FormatNexusTimepix2M.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import absolute_import, division, print_function
+
+import h5py
+
+from dxtbx.format.FormatNexus import FormatNexus
+
+
+class FormatNexusTimepix2M(FormatNexus):
+    @staticmethod
+    def understand(image_file):
+        # Get the file handle
+        with h5py.File(image_file, "r") as handle:
+            if "/entry/instrument/detector/module/data_size" in handle:
+                size = handle["/entry/instrument/detector/module/data_size"]
+                if tuple(size[()]) == (1147, 2069):
+                    return True
+
+        return False
+
+    def __init__(self, image_file, **kwargs):
+
+        super(FormatNexusTimepix2M, self).__init__(image_file, **kwargs)
+
+    def _detector(self):
+        """return a detector with additional masking"""
+
+        detector = super(FormatNexusTimepix2M, self)._detector()
```

Comment:
```suggestion
        super().__init__(image_file, **kwargs)

    def _detector(self):
        """return a detector with additional masking"""

        detector = super()._detector()
```
Python 3 doesn't need extra magic spells